src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -56,19 +62,48 @@ export default class ReplyField extends React.Component<Props, State> {
         prevEditorState = EditorState.forceSelection(prevEditorState, selectionState);
 
         this.state = {
+            activeItemIndex: 0,
             editorState: prevEditorState,
             popupReference: null,
         };
     }
 
     componentDidMount(): void {
         this.focusEditor();
+        // restore PopupList if it was active
+        this.updatePopupReference();
```

Comment:
When would this occur?

Reply:
> When would this occur?

zoom in/out

Reply:
But it also has popper position problem now. I'll remove in this pr and address popper issue in an another pr.